src/components/TabItem/TabItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Tab } from '../../Types/Type';
+
+type Props = {
+  tab: Tab
+  setSelektedTab: (x: Tab) => void
+};
+
+export const TabItem: React.FC<Props> = ({
+  tab,
+  setSelektedTab,
+}) => {
+  return (
+    <a
+      href={`#${tab.id}`}
```

Comment:
you can use shorthand return here
```
() => (
  returned elements here
)
```

Reply:
![image](https://github.com/mate-academy/react_tabs/assets/122989585/1b631fa4-fbe3-4f4e-bf4a-0bbca09c37b2)